packages/governance/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
   "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
Looking at [the documentation for `allowSyntheticDefaultImports`](https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports), it seems like a bad practice to depend on it, and we should probably fix our code instead.

Did that turn out to be too much work, or would you like to address that change in a future PR?  Otherwise, I'd be curious about your experiences trying to turn this option off.

Reply:
Most of them were imports of 3rd party npm packages or NodeJS built-in modules.

Something we want to move away from but out of scope of this PR. The jsconfig filename was always enabling it. At least now it's only enabled in packages that fail without it. We can burn them down over time.